website/public/js/controllers/groupsCtrl.js

```diff
@@ -173,6 +173,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           member.petCount = Shared.countPets($rootScope.countExists(member.items.pets), member.items.pets);
           member.mountCount = Shared.countMounts($rootScope.countExists(member.items.mounts), member.items.mounts);
         $scope.profile = member;
+        $scope.e = member.items.gear.equipped;
```

Comment:
Actually, a refactor to add scope variables to both wouldn't be complicated at all; not sure why I thought it would be. The variables would just need to be initialized [here](https://github.com/HabitRPG/habitrpg/blob/39cb9bfb990d883a794cdc4d514aff6765c0b0c8/website/public/js/controllers/userCtrl.js#L5) and [here](https://github.com/HabitRPG/habitrpg/blob/39cb9bfb990d883a794cdc4d514aff6765c0b0c8/website/public/js/controllers/groupsCtrl.js#L175).